.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## Description
+
+Thank you for your contribution to the PySyft repository.
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
+List any dependencies that are required for this change.
+
+If your pull request closes a GitHub issue, then set its number below.
+
+Fixes # (issue)
+
+## Type of change
+
+Please mark options that are relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```

Comment:
It could be useful to have a "Notebooks" option as notebook-only PRs don't neatly fit into these options

Reply:
I guess they would fit into New feature. By notebooks only do you mean something in the lines of changes to tutorials? If required I can add an option such as "Added/Modified a notebook in examples".

Reply:
Having an entry for tutorials here would be good.